bloom-housing/bloom
# refactor: move authentication helpers out of ui-components

# Pull Request Template

## Issue Overview

This PR addresses #2771


## Description

Removes authentication helpers out of ui-components, as they are Bloom-specific and not relevant for consumers of the component library.

## How Can This Be Tested/Reviewed?

The only code change was in the terms component, which needed a new prop set in order to be able to remove some of the auth dependencies, so ensure that when adding a new partners user, the terms flow works as expected.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
